cli/command/formatter/history_test.go

```diff
@@ -0,0 +1,178 @@
+package formatter
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/pkg/stringid"
+	"github.com/docker/docker/pkg/stringutils"
+)
+
+type historyCase struct {
+	historyCtx historyContext
+	expValue   string
+	expHeader  string
+	call       func() string
+}
+
+func TestHistoryContext_ID(t *testing.T) {
```

Comment:
The granularity of the added tests seems different from existing tests.
(Please refer to existing `*_test.go`)